python/lsst/cp/pipe/ptc.py

```diff
@@ -76,7 +78,7 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
     )
     makePlots = pexConfig.Field(
         dtype=bool,
-        doc="Plot the PTC curves?.",
+        doc="Plot the PTC curves?",
```

Comment:
This is a trivial comment, but I thought that docstrings needed to be full sentences that ended with periods. Why did you delete the period?

Reply:
A question mark takes the role of a period. There should never be a period after a question mark.

Reply:
Thanks for the English explanation, that is indeed the reason. There is loads of prior art where boolean config options which contain an imperative verb (`do*`, `make*` _etc_) have docstrings ending in a question mark, so will leave this as-is.